libr/anal/Makefile

```diff
@@ -4,7 +4,7 @@ EXTRA_TARGETS+=do
 EXTRA_CLEAN=doclean
 
 NAME=r_anal
-DEPS=r_util r_reg r_syscall
+DEPS=r_util r_reg r_syscall r_asm
```

Comment:
cant you just link against those .o? because r_asm can be compiled without xtensa support and this will break the build

> On 01 Sep 2016, at 14:29, Vlad Ivanov <notifications@github.com> wrote:
> 
> In libr/anal/Makefile <https://github.com/radare/radare2/pull/5652#discussion_r77163715>:
> 
> > @@ -4,7 +4,7 @@ EXTRA_TARGETS+=do
> >  EXTRA_CLEAN=doclean
> >  
> >  NAME=r_anal
> > -DEPS=r_util r_reg r_syscall
> > +DEPS=r_util r_reg r_syscall r_asm
> It's where xtensa_* stuff is
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/radare/radare2/pull/5652/files/2ad9e5d0871b3c1d1ca5c06bcc89856b194b94e5#r77163715>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AA3-lpQYJz4rLvOFbK3Two8V1nSp34hWks5qlsUXgaJpZM4JwqCS>.
>